src/pydvl/influence/frameworks/twice_differentiable.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC
+from typing import Callable, Generic, List, Sequence, Tuple, TypeVar
+
+TensorType = TypeVar("TensorType", bound=Sequence)
+ModelType = TypeVar("ModelType")
+
+
+class TwiceDifferentiable(ABC, Generic[TensorType, ModelType]):
+    def __init__(
+        self,
+        model: ModelType,
+        loss: Callable[[TensorType, TensorType], TensorType],
```

Comment:
The more I look at this, the more I wonder, why does something that computes second derivatives of functions need to know about models and losses? The thing that is differentiated is the composition L(f), evaluated at x,y, wrt. the parameters of f, correct? I would expect this wrapper to wrap around a single function, which would be anything. I would then use some `compose` primitive to compose models, losses, whatever. Maybe we can make an issue for this? Or does it not make sense?

Reply:
See #315